modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -1214,7 +1259,7 @@ void ONNXImporter::parseReduce(LayerParams& layerParams, const opencv_onnx::Node
         node_proto.set_input(0, node_proto.output(0));
         node_proto.set_output(0, output_name);
     }
-    else if (!layerParams.has("axes") && (layer_type == "ReduceMean" || layer_type == "ReduceSum" || layer_type == "ReduceMax"))
+    else if (!layerParams.has("axes") && layer_type !=  "GlobalMaxPool" && layer_type !=  "GlobalAveragePool")
```

Comment:
remove extra spaces after `!=`(should be just one)

Reply:
Thanks for the detailed code review, I learned a lot from your comments. All errors are fixed.